doc/cookbook/splitting.md

```diff
@@ -133,116 +133,91 @@ the split files.
   $ pachctl put file users@master -f user_data.txt --split line --target-file-bytes 100
   ```
 
-## Specifying a Header or Footer
+## Specifying a Header
 
-Additionally, if your data has a common header or footer, you can specify them
-manually by using `pachctl put-header` or `pachctl put-footer`. This
-functionality is helpful for CSV data.
+If your data has a common header, you can specify it
+manually by using `pachctl put file` with the `--header-records` flag.
```

Comment:
is the --header-records flag used for SQL data as well? Is it still used to specify number of lines for the header?

Reply:
Actually, I think it's `--split=(json|line|csv)` only according to help. Let me fix that.